pb/lexers/toml.py

```diff
@@ -24,12 +24,21 @@ class TOMLLexer(RegexLexer):
 
     tokens = {
         'root': [
+            # heading
+            (r'^(\s*)(\[+)(.*?)(\]+)$', bygroups(Text, Text, Keyword, Text)),
 
             # Basics, comments, strings
-            (r'\s+', Text),
+            (r'\s+$', Text),
```

Comment:
probably trailing whitespace should be un-tokenized, but that might be too opinionated.